build/makelib/common.mk

```diff
@@ -51,19 +47,38 @@ ifeq ($(PLATFORM),linux_arm64)
 CROSS_TRIPLE = aarch64-linux-gnu
 endif
 ifeq ($(PLATFORM),darwin_amd64)
+# force the build of a linux binary when running on MacOS
+GOHOSTOS=linux
 CROSS_TRIPLE=x86_64-apple-darwin15
 endif
 ifeq ($(PLATFORM),windows_amd64)
+# force the build of a linux binary when running on Windows
+GOHOSTOS=linux
```

Comment:
Should `GOHOSTOS=linux` be set always, for example on line 60 next to the other vars?

Reply:
I suppose this should work yes.